packages/manager/src/constants/API_ENDPOINTS.ts

```diff
@@ -102,6 +105,7 @@ If you didn't intend to run Slice Machine this way, stop it immediately and unse
 				PrismicUnsplash: "https://unsplash.prismic.io/",
 				SliceMachineV1: "https://sm-api.prismic.io/v1/",
 				RepositoryService: "https://repository.prismic.io/",
+				LocaleService: "https://locale.prismic.io/",
```

Comment:
⚠️ Did you test that? I don't understand this related to Michel communication. It should be `"https://locale.internal.prismic.io/"`, correct?

Reply:
I got a little bit confused with the repository url one line up that don't have the internal. added.
Just tested and it seems that both options (with and without internal) work at the moment.
Will add that (should I also add it for repository service?)

Reply:
fixed